$MirOS: ports/lang/egcs/gcc4.4/patches/patch-configure_ac,v 1.1 2009/12/05 15:38:53 tg Exp $
--- configure.ac.orig	Sat Apr 25 04:10:05 2009
+++ configure.ac	Sun Dec  6 14:52:48 2009
@@ -305,7 +305,7 @@ if test "${ENABLE_GOLD}" = "yes"; then
   case "${target}" in
     *-*-elf* | *-*-sysv4* | *-*-unixware* | *-*-eabi* | hppa*64*-*-hpux* \
     | *-*-linux* | frv-*-uclinux* | *-*-irix5* | *-*-irix6* \
-    | *-*-netbsd* | *-*-openbsd* | *-*-freebsd* | *-*-solaris2*)
+    | *-*-mirbsd* | *-*-netbsd* | *-*-openbsd* | *-*-freebsd* | *-*-solaris2*)
       case "${target}" in
         *-*-linux*aout* | *-*-linux*oldld*)
           ;;
@@ -363,6 +363,9 @@ case "${host}" in
   *-*-cygwin*)
     noconfigdirs="$noconfigdirs autoconf automake send-pr rcs guile perl"
     ;;
+  *-*-mirbsd*)
+    noconfigdirs="$noconfigdirs intl zlib"
+    ;;
   *-*-netbsd*)
     noconfigdirs="$noconfigdirs rcs"
     ;;
@@ -485,6 +488,16 @@ case "${target}" in
     skipdirs="$skipdirs zlib fastjar target-libjava target-boehm-gc target-zlib"
     noconfigdirs="$noconfigdirs target-libgloss"
     ;;
+  *-*-mirbsd*)
+    noconfigdirs="$noconfigdirs target-newlib target-libgloss"
+# no libgcj for now
+#    case "${target}" in
+#      i*86-*-*) ;;
+#      *)
+	noconfigdirs="$noconfigdirs ${libgcj}"
+#	;;
+#    esac
+    ;;
   *-*-netbsd*)
     # Skip some stuff on all NetBSD configurations.
     noconfigdirs="$noconfigdirs target-newlib target-libiberty target-libgloss"
@@ -1106,6 +1119,9 @@ case "${host}" in
     ;;
   *-*-sysv4*)
     host_makefile_frag="config/mh-sysv4"
+    ;;
+  i?86-*-mirbsd*)
+    # Do not use -O2 -g -fomit-frame-pointer, may be broken.
     ;;
   # This is placed last to prevent interfering with the cases above.
   i[[3456789]]86-*-*)
